Exercises/4-count-types.js

```diff
@@ -1,5 +1,11 @@
 'use strict';
 
-const countTypesInArray = null;
-
+const countTypesInArray = array => {
+  const hash = {};
+  for (const value of array) {
+    const type = typeof value;
+    hash[type] ? hash[type]++ : hash[type] = 1;
```

Comment:
Try to rewrite, make it simple or see implementations here: https://github.com/HowProgrammingWorks/DataTypes/pull/21#discussion_r324387734